doc/source/deploy/index.rst

```diff
@@ -1,55 +1,277 @@
+and MongoDB query analytics, run:
+
+.. code-block:: bash
+
+   $ sudo pmm-admin --dev-enable add mongodb
+
+.. note:: MongoDB query analytics is experimental
+   and requires the ``--dev-enable`` option when adding.
+   Without this option, only general system metrics and MongoDB metrics
+   are added.
+
+To enable ProxySQL performance metrics, run:
+
+.. code-block:: bash
+
+   $ sudo pmm-admin add proxysql:metrics
+
+To see what is being monitored, run:
+
+.. code-block:: bash
+
+   $ sudo pmm-admin list
+
+For example, if you enable general OS and MongoDB metrics monitoring,
+the output should be similar to the following:
+
+.. code-block:: text
+
+   $ sudo pmm-admin list
+
+   ...
+
+   PMM Server      | 192.168.100.1
+   Client Name     | ubuntu-amd64
+   Client Address  | 192.168.200.1
+   Service manager | linux-systemd
 
-#. :ref:`Start data collection <start-collect>`
+   ---------------- ----------- ----------- -------- ---------------- --------
+   SERVICE TYPE     NAME        LOCAL PORT  RUNNING  DATA SOURCE      OPTIONS
+   ---------------- ----------- ----------- -------- ---------------- --------
+   linux:metrics    mongo-main  42000       YES      -
+   mongodb:metrics  mongo-main  42003       YES      localhost:27017
 
-Upgrading
-=========
+For more information about the available commands of :program:`pmm-admin add`,
+run :program:`pmm-admin add --help` in your terminal.
 
-To upgrade PMM:
+.. _deploy.updating:
 
-1. :ref:`Upgrade PMM Server <upgrade-server>`
+Updating
+================================================================================
 
-#. :ref:`Upgrade PMM Client <upgrade-client>`
-   on all hosts that you are monitoring
+When changing to a new version of |product-abbrev|, you update the
+|product-abbrev| server and each |product-abbrev| client separately.
+
+The updating procedure of your |product-abbrev| server, depends on the option
+that you selected for installing it. If you have isntalled your |product-abbrev|
+server from a :program:`docker` image, follow instruction in the
+:ref:`update-server.docker` section.
+
+If you are running *PMM Server* as a :ref:`virtual appliance <run-server-ova>`
+or using :ref:`Amazon Machine Image <run-server-ami>`, you can use the update
+button in the bottom right corner of the |product-abbrev| home page (see
+:term:`PMM Home Page`).
+
+.. figure:: ../images/update-button.png
+
+   Update your server by clicking the *Update* button on the |product-abbrev|
+   landing page.
+
+.. rubric:: Updating |product-abbrev| clients
+
+When a newer version of *PMM Client* becomes available, you can update to it
+from the Percona software repositories:
+
+* For Debian or Ubuntu::
+
+   $ sudo apt-get update && sudo apt-get install pmm-client
+
+* For RedHat or CentOS::
+
+   $ yum update pmm-client
+
+If you have installed your |product-abbrev| client manually, you need to remove
+it and then download and install a newer version:
 
 Removing
```

Comment:
We say earlier "Installing clients" - I think we should say "Removing clients" - Removing by itself seems to just  be begging - remove what?